pkg/harvester/models/harvester/persistentvolumeclaim.js

```diff
@@ -30,32 +33,39 @@ export default class HciPv extends HarvesterResource {
   }
 
   get availableActions() {
-    const out = super._availableActions;
+    let out = super._availableActions;
+
     const clone = out.find(action => action.action === 'goToClone');
 
     if (clone) {
       clone.action = 'goToCloneVolume';
     }
 
+    if (this.storageClass.provisioner !== LONGHORN_DRIVER || this.storageClass.longhornVersion !== DATA_ENGINE_V2) {
```

Comment:
Could you revert the change in persistentvolumeclaim.js ? 
we need keep take snapshot for LHv2 volume.